<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Sylfaen</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>CONTENT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="TE_Content"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Sylfaen</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>DATE       </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCalendarWidget" name="Calendar">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="gridVisible">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QCalendarWidget::SingleSelection</enum>
              </property>
              <property name="horizontalHeaderFormat">
               <enum>QCalendarWidget::ShortDayNames</enum>
              </property>
              <property name="verticalHeaderFormat">
               <enum>QCalendarWidget::NoVerticalHeader</enum>
              </property>
              <property name="navigationBarVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>SLEEP TIME</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QTimeEdit" name="Time_W"/>
              </item>
              <item row="1" column="1">
               <widget class="QTimeEdit" name="Time_S"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>WAKE TIME</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>WEATHER</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="CB_Weather">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>100</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>Sunny</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Cloudy</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Rainy</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Snowy</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Foggy</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Day of Diary</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="CB_SavedDiary"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="BTN_Save">
        <property name="font">
         <font>
          <family>Sylfaen</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
